packages/bigquery/test/index.js

```diff
@@ -143,14 +143,38 @@ describe('BigQuery', function() {
       assert(bq instanceof Service);
 
       var calledWith = bq.calledWith_[0];
-
       var baseUrl = 'https://www.googleapis.com/bigquery/v2';
-      assert.strictEqual(calledWith.baseUrl, baseUrl);
+      assert.strictEqual(bq.baseUrl, baseUrl);
```

Comment:
This change means we are relying on behavior from `Service`, which we are trying to isolate from for the unit tests. Let's check:

- `calledWith.defaultApiEndpoint`
- `calledWith.environmentVariables`
- `calledWith.basePath`

to make sure we had a proper instantiation of Service.

Reply:
Fair enough, I'll make the changes. It should also make the final merge smaller which is a good thing IMO